server/src/domain/asset/dto/map-marker.dto.ts

```diff
@@ -31,4 +31,10 @@ export class MapMarkerDto {
   @IsBoolean()
   @Transform(toBoolean)
   withPartners?: boolean;
+
+  @ApiProperty()
```

Comment:
This should not be needed

Reply:
:+1:

The class got moved to `server/src/dtos/search.dto.ts` and doesn't have this annotation anymore.